### PR TITLE
Raw type

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
This suppress warnings and even remove some casting.
Some warnings remains and I find no trivial way to deal with them

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/ui/ActionBarOverflow.java [hunk 1]
```diff
@@ -113,11 +115,11 @@ static boolean hasUsableMethod() {
 
     @CheckResult
     @VisibleForTesting(otherwise = VisibleForTesting.NONE)
-    static Pair<Class, Method> getPrivateMethodOnlyHandleExceptions(String className, String methodName) {
+    static Pair<Class<?>, Method> getPrivateMethodOnlyHandleExceptions(String className, String methodName) {
         Method action = null;
         Class<?> menuItemImpl = null;
         try {
-            menuItemImpl = Class.forName(className);
+            menuItemImpl = forName(className);
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/ui/ActionBarOverflow.java:122]: I don't think this has value
```suggestion
            menuItemImpl = Class.forName(className);
```

File: AnkiDroid/src/main/java/com/ichi2/ui/ActionBarOverflow.java [hunk 2]
```diff
@@ -10,6 +10,8 @@
 import androidx.annotation.VisibleForTesting;
 import timber.log.Timber;
 
+import static java.lang.Class.forName;
+
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/ui/ActionBarOverflow.java:14]:
```suggestion
```

matching comment below, why static import a method of an object we already have in namespace? Makes code below less clear I believe

**Review by mikehardy — APPROVED** [review]
tentatively approving module the removal of the static-import of Class.forName, for me

**Review by mikehardy — APPROVED** [review]

**Comment by mikehardy** [comment]
```
        Caused by:

        com.ichi2.utils.JSONException: JSONArray initial value should be a string or collection or array.

            at com.ichi2.utils.JSONArray.<init>(JSONArray.java:139)

            at com.ichi2.libanki.Models.moveField(Models.java:585)

            at com.ichi2.libanki.ModelTest.test_fields(ModelTest.java:89)


```

**Comment by mikehardy** [comment]
@Arthur-Milchior needs fix for test failure

**Comment by mikehardy** [comment]
@Arthur-Milchior took more conflicts :laughing:

**Comment by Arthur-Milchior** [comment]
Conflict removed

**Review by david-allison-1 — APPROVED** [review]

**Pull request merged**